MeshLib/MeshSubset.h

```diff
@@ -53,7 +53,24 @@ class MeshSubset
     /// \param vec_items Vector of Node pointers.
     MeshSubset(const Mesh& msh, std::vector<Node*> const& vec_items)
         : _msh(msh), _nodes(vec_items)
-    {}
+    {
+        // testing if the given nodes belong to the mesh.
+        auto node_is_part_of_mesh =
+            [mesh_nodes = _msh.getNodes()](MeshLib::Node* const& n) {
+                auto it = std::find(begin(mesh_nodes), end(mesh_nodes), n);
```

Comment:
This is a pointer comparison. Is that correct?

Reply:
Yes. Just a logical check (not geometrical) if the subset of nodes is indeed a part of the mesh nodes vector.